pkg/server/source/github.go

```diff
@@ -95,8 +102,12 @@ func (g *gitHubSourceSystem) MintAccessToken(ctx context.Context, org, repo stri
 	}
 	tokenRequest := githubauth.TokenRequest{
 		Repositories: repositories,
-		Permissions:  permissions,
+		Permissions:  map[string]string{},
```

Comment:
check my understanding, if this is empty, then zero permissions are given, but if the permissions attribute was missing altogether, then all permissions are given correct?

Reply:
If the map is empty then we get all permissions defined on the app. To get to this point we have multiple checks to verify that it is allowed to request that.

If the `Permissions` attribute is null then GitHub throws a 500